### DISCUSSION TITLE
How to install the standalone debugger

### DISCUSSION THREAD

**Discussion by JackNWhite** [eclipse-cdt/cdt#708] [Q&A]
Hello,

I would like to use the standalone debugger, but the instructions do not and have never worked for me. There used to be, instead of the install script, a debugger package that was actually standalone; this was a better arrangement, because it worked.

Once I have run the install script in the plugin scripts directory, there is a problem with the cdtdebug.sh script, but this is easily solved by changing the shebang line at the top of the file to use bash, rather than sh.

After that, Eclipse begins to start, but then produces a long list of unresolved modules in the log, and quits.

This attempt was made using the debugger plugin in `org.eclipse.cdt.debug.application_1.2.700.202204200013`.

[1708090905236.log](https://github.com/eclipse-cdt/cdt/files/14311714/1708090905236.log)

I have another copy of the plugin (`org.eclipse.cdt.debug.application_11.4.0.202311211857`).

The outcome is same, though with a rather shorter log.
[1708091247553.log](https://github.com/eclipse-cdt/cdt/files/14311746/1708091247553.log)

Is there a simpler, failsafe way to install the debugger? Perhaps someone else is now packaging it?

**Comment by jonahgraham** [comment]
This is a known issue - there is some discussion on this in https://github.com/eclipse-cdt/cdt/issues/591

> Is there a simpler, failsafe way to install the debugger?

You can probably build your own version fairly easily.

> Perhaps someone else is now packaging it?

RedHat (IIRC) used to package it, but AFAIU they have stopped doing that. There are some users (and potential users like you) but no one has kept this going and the configuration has bitrotten over time.
